recOrder/scripts/samples.py

```diff
@@ -0,0 +1,67 @@
+import shutil
+from pathlib import Path
+
+from iohub import open_ome_zarr
+from napari.utils.notifications import show_warning
+from platformdirs import user_data_dir
+from wget import download
+
+
+def download_and_unzip():  # -> tuple(Path):
+    """Downloads sample data .zip from zenodo, unzips, and returns Paths to the .zarr datasets.
+
+    Skips the download if the files already exist.
+
+    Uses platformdirs.user_data_dir to store data.
+    """
+    temp_dirpath = Path(user_data_dir("recOrder"))
+    temp_dirpath.mkdir(exist_ok=True)
+    data_dirpath = temp_dirpath / "sample_contribution"
```

Comment:
Should this be version-specific so that it can be updated?

Reply:
Good idea! Added v1.4 to folder name